docs/home/SECURITY.md

```diff
@@ -150,74 +154,85 @@ The list does not include ports that are bound to the loopback interface (localh
 ### Connection protocols and ciphers used by components in Epiphany
 
 Below you can find list of cipersuites and protocols used for communication set in Epiphany on per component basis. This
-doesn't contains information about deprecated components: Apache Ignite, Istio, Hashicorp Vault.
+doesn't contain information about deprecated components: Apache Ignite, Istio, Hashicorp Vault.
```

Comment:
![image](https://user-images.githubusercontent.com/6271862/147463820-7639e413-e176-4a33-94d0-aff98c6c21a7.png)

Reply:
This is about 'deprecated' components

Reply:
Changed to potentially deprecated.